modules/images-configuration-create-config-map.adoc

```diff
@@ -0,0 +1,60 @@
+id="steps-for-manual-configmap-creation"]
+= Steps create the image signature configmap
+
+The following commands will create and apply the image signature configmap to the cluster you want to update. 
+
+.Procedure
+
+. Find the link:https://access.redhat.com/solutions/4583231[version] you want the cluster to update.
+
+. Add the version to OCP_RELEASE_NUMBER environment variable.
++
+----
+$ export OCP_RELEASE_NUMBER=<{product-version}.0>
+----
++
+. Get the release image digest from link:https://quay.io/[Quay].
++
+----
+$ export DIGEST="$(oc adm release info quay.io/openshift-release-dev/ocp-release:${OCP_RELEASE_NUMBER}-x86_64 | sed -n 's/Pull From: .*@//p')"
+----
++
+. Get the image signature from link:https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release[mirror.openshift.com] website.
++
+----
+$ export SIGNATURE_BASE64=$(curl -s "https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release/$(echo $DIGEST | cut -d: -f1)=$(echo $DIGEST | cut -d: -f2)/signature-1" | base64 -w0 && echo)
+----
++
+. Get the digest algorithm.
++
+----
+export DIGEST_ALGO=$(echo $DIGEST | cut -d: -f1)
```

Comment:
I [still][1] think this and `DIGEST_SIGNATURE` should be shifted up above the `curl` so you don't have to inline the `$(...)` command substitution in the `curl` line.

[1]: https://github.com/openshift/openshift-docs/pull/21993#discussion_r423305654